include/fmt/os.h

```diff
@@ -59,6 +59,14 @@
 #  endif
 #endif
 
+#ifdef __MINGW32__
+// The following will be undefined if the _POSIX_ flag is set.
+// For the rest of the functions, we'll use the underscore prefix.
+#  ifndef _fdopen
+#     define _fdopen fdopen
```

Comment:
We shouldn't define any global macros. Perhaps the `#ifdef` should go to https://github.com/fmtlib/fmt/blob/8069265373f999ba31dd200e7dd090104a4236ad/src/os.cc#L292 and choose between `FMT_POSIX_CALL` and `FMT_SYSTEM` based on whether `_POSIX_` is defined.

Reply:
Makes sense. I've updated the commit accordingly.